poetry.lock

```diff
@@ -1,3 +1,35 @@
+[[package]]
```

Comment:
See pyproject.toml

Reply:
Since it is generated by poetry according to pyproject.toml.